src/behaviours/use-drag.ts

```diff
@@ -383,15 +384,18 @@ export function useDrag(
     }
 
     function onPanSessionStart(event: MouseEvent | TouchEvent | PointerEvent) {
-        // Prevent browser-specific behaviours like text selection or Chrome's image dragging.
         if (
             event.target &&
             !allowDefaultPointerDown.has((event.target as Element).tagName)
         ) {
-            event.preventDefault()
-            // Make sure input elements loose focus when we prevent the default.
-            if (document.activeElement instanceof HTMLElement) {
-                document.activeElement.blur()
+            if (!supportsTouchEvents()) {
```

Comment:
Could you add a note as to why we're adding this text (describe the bug we're avoiding)